cli/server/server.go

```diff
@@ -337,7 +351,17 @@ Main:
 			shutdownErr = fmt.Errorf("server error: %w", err)
 			cancel()
 
+		case <-rpcServerRestartChan:
```

Comment:
Why can't you listen `sighupCh` here directly?

Reply:
The original design required a separate channel, but it's OK now to use `sighupCh`, so fixed.